scripts/apps/relations/services/RelationsService.ts

```diff
@@ -1,14 +1,20 @@
 import {IArticle} from 'superdesk-interfaces/Article';
 
-RelationsService.$inject = ['archiveService'];
+RelationsService.$inject = ['archiveService', 'mediaIdGenerator'];
 
-export function RelationsService(archiveService) {
-    this.getRelatedItems = function(item: IArticle) {
+export function RelationsService(archiveService, mediaIdGenerator) {
+    this.getRelatedItems = function(item: IArticle, mediaFields: object) {
```

Comment:
The naming no longer matches the code in my opinion. If the method is called `getRelatedItems`, it should get all of them, including media galleries. If you want to exclude media galleries, rename/create another method like `getRelatedItemsWithoutGalleries` or similar.